_2_oo/src/main/java/code/_3_in_class/Main.java

```diff
@@ -3,6 +3,6 @@
 public class Main {
 
   public static void main(String[] args) {
-    //TODO put your code changes in here
+    System.out.println("Hello world");
   }
 }
```

Comment:
vad ca in cazul tau intellij a modificat 37 de fisiere. 
Sigur ai deschis proiect folosind File -> Open -> "C:\development\workspace\java-training" ?